pkg/interface/src/views/components/Timestamp.tsx

```diff
@@ -24,14 +34,24 @@ const Timestamp = (props: TimestampProps): ReactElement | null => {
   const { hovering, bind } =
     date === true ? { hovering: true, bind: {} } : useHovering();
   let datestamp = stamp.format(DateFormat);
-  if (stamp.format(DateFormat) === moment().format(DateFormat)) {
-    datestamp = 'Today';
-  } else if (
-    stamp.format(DateFormat) === moment().subtract(1, 'day').format(DateFormat)
-  ) {
-    datestamp = 'Yesterday';
+  if (!dateNotRelative) {
```

Comment:
This is probably better handled with `moment.calendar`, instead of special casing the format string like this

Reply:
Does this conflict with your time stamp pr? I like its style because it lets us expose relative time as a setting ppl can choose later.